client/src/modules/break_even_reference/modals/break_even_reference.modal.html

```diff
@@ -0,0 +1,139 @@
+<form name="BreakEvenReferenceForm" bh-submit="BreakEvenReferenceModalCtrl.submit(BreakEvenReferenceForm)" novalidate>
+  <div class="modal-header">
+    <ol class="headercrumb">
+      <li class="static" translate>TREE.ADMIN</li>
+      <li class="title">
+        <span translate>TREE.BREAK_EVEN_REFERENCE</span>
+        <label ng-if="BreakEvenReferenceModalCtrl.isCreating" class="badge badge-warning" translate>FORM.LABELS.CREATE</label>
+        <label ng-if="!BreakEvenReferenceModalCtrl.isCreating" class="badge badge-warning" translate>FORM.LABELS.UPDATE</label>
+      </li>
+    </ol>
+  </div>
+
+  <div class="modal-body" style="overflow-y: scroll; height:300px;">
+    <!-- label -->
+    <div class="form-group" ng-class="{ 'has-error' : BreakEvenReferenceForm.$submitted && BreakEvenReferenceForm.description.$invalid }">
```

Comment:
This should be `BreakEvenReferenceForm.label.$invalid` instead of `BreakEvenReferenceForm.description.$invalid`.  Otherwise, it won't show validation correctly:
![CoolSongBro](https://user-images.githubusercontent.com/896472/54202472-659a2d00-44d0-11e9-8d69-15e2cba14215.PNG)